src/contexts/ProfitAndLossComparisonContext/ProfitAndLossComparisonContext.tsx

```diff
@@ -18,4 +19,10 @@ export const PNLComparisonContext = createContext<PNLComparisonContextType>({
   refetch: function (): void {
     throw new Error('Function not implemented.')
   },
+  getProfitAndLossComparisonCsv: function (): Promise<{
+    data?: S3PresignedUrl | undefined
```

Comment:
Not sure if having both `?` and `undefined` is necessary... however I already experienced some odd behavior and necessity to use both

```suggestion
    data?: S3PresignedUrl
```